slides/06_publication.qmd

```diff
@@ -222,8 +223,11 @@ Maintainer: 'Friedrich Pahlke'
 
 # Exercise
 
+Use the same repository created from template https://github.com/kkmann/simulatr
+
 - Run CRAN like checks in RStudio: use `--as-cran` in check options
 - Look at the CI/CD checks reported on your GitHub repository
   - Where are the `R CMD check` results?
-- Deploy the `pkgdown` website with `usethis`
-
+- Deploy the website with `pkgdown`
```

Comment:
can we still reference `usethis` as alternative? that can be preferred by some participants/teachers

Reply:
here if we are not involving much with the github actions, then we have to use `pkgdown::deploy_to_branch` functionality.
`usethis::use_pkgdown` set up the _pkgdown.yaml file, `usethis::use_pkgdown_github` set up github repository to use `gh-pages` be the site etc, `usethis::use_github_action` to enable the github action to deploy the website. If we are going to set up the github actions then we can still use `usethis` to do that

Reply:
Yeah I think here we can mention both options, via pkgdown and via usethis